Foundation/URLAuthenticationChallenge.swift

```diff
@@ -192,3 +192,21 @@ open class URLAuthenticationChallenge : NSObject, NSSecureCoding {
         }
     }
 }
+
+extension _HTTPURLProtocol : URLAuthenticationChallengeSender {
```

Comment:
We must add `NSUnimplemented()` or `fatalError()` with a message to these empty methods.

Reply:
Done ..Thanks